scp/base_fluid.py

```diff
@@ -129,6 +129,21 @@ def _check_temperature(self, temp: float) -> float:
         else:
             return temp
 
+    @abstractmethod
+    def freeze_point(self, x: float) -> float:
```

Comment:
Got it, so this is a nice new function that can be called even from the CLI to get the freezing point of the mixture for a given concentration.  :+1: